# user_msg.msg
string name
int32 age
